### PR TITLE
[POC] Remove bonzo and qwery

### PR DESCRIPTION
**Pull request by joelochlann**
Following on from discussion in #17425. Here are just the changes that involve removing `$.create` and `.insertAfter()` and replacing them with our own library functions

Could be a useful reference point for future discussions. But feel free to close it if you feel it's cluttering up the list of open PRs.

### CODE REVIEW

**Comment by gustavpursche** [comment]
Just my 5ct: I'd like to rather not start with this now until we have a longer discussion about the future we want. It feels a bit, that we'd invent a second bonzo over time, if we start to wrap everything into `dom.js`. I like to encourage people to use native stuff and to provide helpers, for the hard things (event delegation, inserts and create). Also right now I don't really see the problem it would solve - as part of the ES6 conversion we got rid of bonzo in many places and replaced bean by native bindings. This is an effort which could continue, so that in the end we only have a handful of problemats, that we could solve with a helper like you proposed, but much more specialized.

@sndrs and @SiAdcock are doing a lot of thinking about the [future of future](https://github.com/guardian/frontend-future) and there is already a [PR in frontend](https://github.com/guardian/frontend/pull/17141) trying to implement it. I guess depending on the progress there, we know what the priorities and directions are for the next quarter(s).

What do you think?

**Comment by sndrs** [comment]
i feel like i can see and agree with everyone's points simultaneously here, which is not that useful...

@joelochlann would you be able to outline what prompted you to do this in the first place? i think we all feel slightly icky about those 3 micro libs, but if we can elucidate clearly what gives rise to those feelings, it might help us decide what it is we'd rather see?

**Comment by sndrs** [comment]
also, let's move the chat to #17467 – keep the PR list clearer

File: static/src/javascripts/lib/dom.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+// @flow
+
+const insertAfter = (
+    elementToInsert: Element,
+    elementToInsertAfter: Element
+): void => {
+    const parent = elementToInsertAfter.parentNode;
```
Comment (regiskuckaertz) [inline static/src/javascripts/lib/dom.js]: These 10 lines become `elementToInsertAfter.insertAdjacentElement('afterend', elementToInsert)`. Same for the `insertBefore`: `elementToInsertBefore.insertAdjacentElement('beforebegin4', elementToInsert)`.

Comment (joelochlann) [inline static/src/javascripts/lib/dom.js]: Thanks! That's a lot better - didn't know about `.insertAdjacentElement()`

**Pull request closed without merging**